src/pages/VideoCall/VideoCallFrame.tsx

```diff
@@ -45,7 +72,36 @@ const VideoCallFrame = ({
                 <div className="waiting-participant">{participantStatus}</div>
             )}
 
-            {hasLeft && <Redirect to="/" />}
+            {redirectToRoot && <Redirect to="/" />}
+
+            {hasLeft && (
+                <div>
+                    <Card onClick={(val) => setRedirectToRoot(val)} />
+                    <Grid
+                        container
+                        justify="center"
+                        alignItems="center"
+                        className={classes.root}>
+                        <Typography
+                            style={{ textAlign: 'center', marginTop: 20 }}
+                            variant="h5"
+                            component="h6">
+                            Pour nous aider à Améliorer notre service, vous
+                            pouvez noter la qualité de votre visio.
+                        </Typography>
+                    </Grid>
+                    <Grid container justify="center" alignItems="center">
+                        <StyledRating
+                            value={value}
+                            precision={1}
+                            onChange={(event, value) => {
```

Comment:
Inline function is not necessary here. You can group these two functions in one.
```
const onChange = (_, value) => {
  setValue(value)
  addCallRating(value)
}
```

Reply:
it s ok now in the last push.